app/scripts/controllers/transactions/tx-gas-utils.js

```
@@ -139,53 +132,143 @@ export default class TxGasUtil {
+          const receiverBytes = arrayify(txMeta.txParams.to);
+          let receiver16Bytes;
+          if (receiverBytes.length === 16) {
+            receiver16Bytes = receiverBytes;
+          } else if (receiverBytes.length === 32) {
+            // 32-byte format: [16 zeros][16 real address] - extract last 16
+            receiver16Bytes = receiverBytes.slice(16);
+          } else if (receiverBytes.length === 20) {
+            // 20-byte Ethereum-style: take last 16 bytes
+            receiver16Bytes = receiverBytes.slice(4);
+          } else {
+            log.warn(`VM2 gas estimate: Unexpected receiver address length: ${receiverBytes.length}`);
```

ESLint

[error] Replace ``VM2·gas·estimate:·Unexpected·receiver·address·length:·${receiverBytes.length}`` with `⏎··············`VM2·gas·estimate:·Unexpected·receiver·address·length:·${receiverBytes.length}`,⏎············`

(prettier/prettier)